_docs-2/troubleshooting/advanced.md

```diff
@@ -333,12 +333,12 @@ overridden by using the `--local-wal-directories` option on the tool. It can be
 **I am trying to start the manager after upgrading but the upgrade is aborting with the following message:**
   `org.apache.accumulo.core.client.AccumuloException: Aborting upgrade because there are outstanding FATE transactions from a previous Accumulo version.`
 
-You can use the shell to delete completed FATE transactions using the following:
+You can use the admin fate command to delete completed FATE transactions using the following:
 
 * Start tservers
 * Start shell
-* Run `fate print` to list all
-* If completed, just delete with `fate delete`
+* Run `accumulo admin fate --print` to list all transactions
+* If the transactions have completed, just delete with them with `accumulo admin fate --delete TXID [TXID...]`
```

Comment:
Does this work in all 2.x versions? These docs cover all 2.x. If it only works for a subset of these versions, it should be called out, as in "Prior to 2.x.y, the command was in the shell as ..." or similar.

Reply:
It was updated in 2.1 with PR #2914

Reply:
Addressed in f6ec66fb